frontend/src/components/material/DialogMaterialItemEdit.vue

```diff
@@ -13,6 +13,10 @@
       <slot name="activator" v-bind="scope" />
     </template>
 
+    <template #moreActions>
+      <ButtonDelete class="v-btn--has-bg" @click="del" />
```

Comment:
Should we use a confirmation prompt here? Or do we think not necessary because not much damage is done by deleting a material item?

Reply:
I thought it was too cumbersome to delete multiple items. But maybe this is not a good idea consistency wise.

Reply:
[a241a13](https://github.com/ecamp/ecamp3/pull/3797/commits/a241a13907d2671e5b1818c6a9ea6d73d9b4c58e)

Reply:
Thanks. Could you add a `:warning-text-entity="materialItem.article"` to the `PromptEntityDelete`?

Reply:
[ba77541](https://github.com/ecamp/ecamp3/pull/3797/commits/ba77541c4bf467653cc7a0e494544f1a05f789c8)